src/geanyobject.h

```diff
@@ -47,6 +47,7 @@ typedef enum
 	GCB_PROJECT_OPEN,
 	GCB_PROJECT_SAVE,
 	GCB_PROJECT_CLOSE,
+	GDB_PROJECT_BEFORE_CLOSE,
```

Comment:
Yes, this OKAY. The enum values are only used if you use `g_signal_emit()` (instead of `g_signal_emit_by_name()`). Plugins shouldn't be emitting these at all.